src/search_options.hpp

```diff
@@ -91,8 +91,8 @@ struct LambdaOptions : public SharedOptions
     int32_t misMatch      = -3; // only for manual
 
     int32_t  band        = -3;
-    int32_t  minBitScore = -1;
-    double   maxEValue   = 1e-04;
+    int32_t  minBitScore = 42;
+    double   maxEValue   = -1;
     int32_t  idCutOff    = 0;
     uint64_t maxMatches  = 256;
```

Comment:
I forgot to change this to 25

Reply:
This means you need to recreate the search test files I am afraid...

Reply:
No problem